src/PluginsSystem/Microsoft.Performance.Toolkit.Plugins.Core/AvailablePluginInfo.cs

```diff
@@ -18,13 +18,32 @@ public sealed class AvailablePluginInfo
         /// <summary>
         ///     Initializes an instance of <see cref="AvailablePluginInfo"/>.
         /// </summary>
+        /// <param name="metadata">
+        ///     The metadata of the plugin.
+        /// </param>
+        /// <param name="source">
+        ///     The source where this plugin is discovered.
+        /// </param>
+        /// <param name="packageSize">
+        ///     The size, in number of bytes, of the package that makes up this plugin.
+        /// </param>
+        /// <param name="packageUri">
+        ///     The URI to the package that makes up this plugin.
+        /// </param>
+        /// <param name="fetcherResourceId">
+        ///     The resource ID of the fetcher that can be used to download the package.
+        /// </param>
+        /// <param name="publishedOn">
+        ///     The date and time when this plugin was published.
+        /// </param>
         [JsonConstructor]
         public AvailablePluginInfo(
             PluginMetadata metadata,
             PluginSource source,
             ulong packageSize,
             Uri packageUri,
-            Guid fetcherResourceId)
+            Guid fetcherResourceId,
+            DateTimeOffset publishedOn)
```

Comment:
Is DateTimeOffset better than DateTime here?

Reply:
https://stackoverflow.com/questions/4331189/datetime-vs-datetimeoffset

Great read, looks good with DateTimeOffset.